src/main/java/com/spikes2212/utils/ChildNamespace.java

```diff
@@ -0,0 +1,72 @@
+package com.spikes2212.utils;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableValue;
+import edu.wpi.first.wpilibj.Sendable;
+
+import java.util.function.Supplier;
+
+public class ChildNamespace extends RootNamespace {
+
+    private Namespace parent;
+
+    public ChildNamespace(String name, Namespace parent) {
```

Comment:
please add a `String separator` parameter, this will be the character that separates the two names.
add a default constructor that sets the `separator` to be `/`